_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -32,11 +32,11 @@ This rule applies to any element with an `aria-hidden` [attribute value][] of `t
 
 ## Expectation
 
-None of the target elements are [focusable][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [focusable][].
+None of the target elements are part of the [sequential focus navigation][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are part of the [sequential focus navigation][], unless, upon receiving focus, focus is moved to an element that does not fail this rule.
```

Comment:
There is also a full can of worms around what "upon receiving focus" actually means 🙈 
Is it good if the redirection happens after 5 seconds? 0.5s? 0.5ms?
See also [this discussion](https://github.com/act-rules/act-rules.github.io/pull/1444#discussion_r761084251).

Since we somewhat have resolved this point in earlier definition, I'd suggest something like
```suggestion
None of the target elements are part of the [sequential focus navigation][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are part of the [sequential focus navigation][], unless, they are not [focusable][].
```
This is not perfect  since it doesn't look at where the focus is redirect. OTOH it would also accept elements that self-blur (without focusing anything else).